src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/pbs/PBSJobManager.groovy

```diff
@@ -105,7 +73,7 @@ class PBSJobManager extends ClusterJobManager<PBSCommand> {
     @Override
     boolean getDefaultForHoldJobsEnabled() { return true }
 
-    List<String> collectJobIDsFromJobs(List<BEJob> jobs) {
+    private List<String> collectJobIDsFromJobs(List<BEJob> jobs) {
```

Comment:
Why not put this into the base class? and make it public static?

Reply:
Done
Is this method even needed?